### PR TITLE
Restructuring the installation guide to help users choose between modes; Adding the doc for RisingWave in k8s

### LINKED ISSUE #313: Bug: update the risingwave version in installation steps
### Where's the bug?

https://www.risingwave-labs.com/product/

### Describe the bug

We should keep updated the risingwave version in installation command whenever we bump up the stable version.

Screenshot 2022-10-19 at 2 11 05 PM 1

### LINKED ISSUE #331: Bug: the demo repo doesn't start 8 processes.
### Where's the bug?

https://www.risingwave.dev/docs/latest/install-run-connect/

### Describe the bug

 I noticed that there is an inaccurate description in the multi-node cluster section. 
Now it starts 6 processes, and Prometheus is not one of them. Please update the description. 

Also, this sentence needs rewording: "Therefore, it will start 8 processes." "Therefore" is redundant here.

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
 **Description**: 

Click the links to see the preview pages:

  - [Restructuring the installation guide](https://pr-343.d2fbku9n2b6wde.amplifyapp.com/docs/latest/get-started/) to help users choose among installation modes;
  - Adding diagrams and screenshots to make procedures clearer;
  - Tweaking the wording to make the content looks "easier";
  - @arkbriar @Nebulazhang [Adding the doc for RisingWave in k8s](https://pr-343.d2fbku9n2b6wde.amplifyapp.com/docs/latest/risingwave-kubernetes/) ;
  - [Adding buttons in What's RisingWave](https://pr-343.d2fbku9n2b6wde.amplifyapp.com/docs/latest/intro/) to give first-time readers/users directions and prompt them to try RW;
  - Fixing some issues spotted during the bug bash.

 **Related doc issue**: 
Also resolves https://github.com/risingwavelabs/risingwave-docs/issues/313 and https://github.com/risingwavelabs/risingwave-docs/issues/331

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

## Published doc pages
  [ After merging this PR, edit the description to include the links to the updated doc pages here. For example, https://www.risingwave.dev/docs/latest/intro/. ]

### CODE REVIEW

**Review by skyzh — APPROVED** [review]
LGTM except

> Service is automatically terminated after 30 minutes of inactivity.

We should fix this on the kernel side, as this is not the intended behavior.

**Comment by CharlieSYH** [comment]
> LGTM except
> 
> > Service is automatically terminated after 30 minutes of inactivity.
> 
> We should fix this on the kernel side, as this is not the intended behavior.

@skyzh It seems that the Playground is set to exit after sitting idle for 1800 s:

https://github.com/risingwavelabs/risingwave/blob/main/risedev.yml#L133
```
##########################################
  ### Configuration used for ./risedev p ###
  ##########################################

  playground:
    - use: meta-node
      unsafe-disable-recovery: true
      max-idle-secs-to-exit: 1800
    - use: compute-node
    - use: frontend

  playground-test:
    - use: meta-node
      unsafe-disable-recovery: true
      max-idle-secs-to-exit: 1800
      vacuum-interval-sec: 1
      collect-gc-watermark-spin-interval-sec: 1
      min-sst-retention-time-sec: 0
    - use: compute-node
    - use: frontend
    - use: compactor
```

**Comment by skyzh** [comment]
Oh that's true. You might need to wait for the next release before we can fix this.

File: docs/deploy/risingwave-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,284 @@
...
+    ```shell
+    kubectl cluster-info
+    ```
+
+## Deploy the Operator
+
+Before the deployment, ensure that the following requirements are satisfied.
+
+* Docker version ≥ 18.09
+* kubectl version ≥ 1.12
```
Comment (arkbriar) [inline docs/deploy/risingwave-kubernetes.md]: Preferably, let's require the `kubectl` to be `>= 1.18`.

File: docs/deploy/risingwave-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,284 @@
...
+
+* **[Install kubectl](http://pwittrock.github.io/docs/tasks/tools/install-kubectl/)**
+
+    Ensure that the Kubernetes command-line tool [kubectl](https://kubernetes.io/docs/reference/kubectl/) is installed in your environment.
+
+* **[Install psql](https://www.postgresql.org/download/)**
+
+    Ensure that the PostgreSQL interactive terminal [psql](https://www.postgresql.org/docs/current/app-psql.html) is installed in your environment.
+
+* **[Install and run Docker Desktop](https://docs.docker.com/get-docker/)**
```
Comment (arkbriar) [inline docs/deploy/risingwave-kubernetes.md]: The Docker Desktop is only required when the host OS isn't Linux. With Linux, we only need docker.

File: docs/deploy/risingwave-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,284 @@
...
+<TabItem value="clusterip" label="ClusterIP">
+
+By default, the Operator creates a service for the frontend component, through which you can interact with RisingWave, with the type of `ClusterIP`. But it is not accessible outside Kubernetes. Therefore, you need to create a standalone Pod for PostgreSQL inside Kubernetes.
+
+**Steps:**
+
+1. Create a Pod by running the following command.
+
+    ```shell
+    kubectl apply -f examples/psql/psql-console.yaml
```
Comment (arkbriar) [inline docs/deploy/risingwave-kubernetes.md]: Shall we replace the `examples/psql/psql-console.yaml` here with a link?

**Review by arkbriar — APPROVED** [review]
Rest LGTM! Thanks for your efforts!

**Review by hengm3467 — APPROVED** [review]

File: docs/deploy/risingwave-local.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+id: risingwave-local
+title: Run RisingWave locally
+description: Install and run RisingWave locally in your environment.
+slug: /risingwave-local
+---
+
+This article will help you start a RisingWave standalone instance in your environment.
+
+:::caution
+This method starts RisingWave in playground mode, where data is not persisted after the service is terminated.<br/>Start RisingWave in full-featured mode for data persistence and stable performance. <br/>[→ See the comparison](/get-started.md/#run-risingwave)
```
Comment (xiangjinwu) [inline docs/deploy/risingwave-local.md]: `See the comparison` link is broken

**Review by emile-00 — APPROVED** [review]

**Pull request merged**